docker-gunicorn.conf.py

```diff
@@ -0,0 +1,11 @@
+bind = "0.0.0.0:8000"
+worker = 2
+threads = 8
```

Comment:
In the `Procfile` here we have only 4 threads. Would keep the settings:
https://github.com/moneymeets/sepacetamol/blob/master/Procfile